### PR TITLE
Features/garden progress

### PR DESCRIPTION
**Pull request by Br3nda**
I'm entering all this data, to track, predict and plan harvest and planting progress. There's no one page currently to see the progress of plantings. So, i modified the gardens#index to give a quick-glance overview of your gardens.

Before:

image

After:  (note the buttons on hover)
image

Small screen:
before:
image

after:
image

### CODE REVIEW

**Comment by pozorvlak** [comment]
That does look more useful! I'm +1 to this once green, but would appreciate more opinions.

File: app/views/plantings/_card.html.haml [hunk 1]
```diff
@@ -16,7 +16,7 @@
           %dt Owner:
           %dd= link_to planting.owner.login_name, planting.owner
           %dt Garden:
-          %dd= link_to planting.garden.name, planting.garden
+          %dd= link_to planting.garden&.name, planting.garden
```
Comment (pozorvlak) [inline app/views/plantings/_card.html.haml]: TIL about the `&.` operator - nice!

File: app/views/plantings/_progress_bar.html.haml [hunk 2]
```diff
@@ -1,3 +1,9 @@
 .progress
-  %div{ class: "progress-bar progress-bar-#{status}", role: "progressbar", style: "width: #{progress}" }
-
+  - if progress.nil?
+    #{status}
+  - else
+    %div{ class: "progress-bar progress-bar-#{status}", role: "progressbar", style: "width: #{progress}%" }
```
[the remark this answers is not in the document]
Comment (pozorvlak) [reply app/views/plantings/_progress_bar.html.haml]: Hmmm, I'm not sure how we can achieve the desired effect without using inline style attributes here. I'd be OK with disabling this warning.

**Comment by Br3nda** [comment]
Excellent

Once the progress bar was more visible, it has exposed to me how many bugs we have in that code. Eg it's only updating the progress on the 2nd time your edit a planting. So once this is merged I want to look at massively improving predictions.

**Comment by Br3nda** [comment]
There's some tweaks to alignment I'll do here as well.

**Comment by pozorvlak** [comment]
Sorry, I missed that you wanted to do more work on this branch.

**Comment by Br3nda** [comment]
that's cool! onwards we go.

**Pull request merged**